src/redux/books/categories.js

```diff
@@ -0,0 +1,12 @@
+const categoriesReducer = (state = [], action) => {
+  if (action.type === 'checkStatus') {
```

Comment:
- [ ] Kindly create a categories directory and move the `categories.js` file from books directory to the newly created categories directory because it is required to have the `categories.js` file inside the categories directory.